crates/test_utils/wasm/wasm_workspace/agent_info/src/lib.rs

```diff
@@ -2,5 +2,5 @@ use hdk3::prelude::*;
 
 #[hdk_extern]
 fn agent_info(_: ()) -> ExternResult<AgentInfo> {
-    Ok(hdk3::prelude::agent_info()?)
+    hdk3::prelude::agent_info()
```

Comment:
there's no conversion because `HdkResult` no longer exists as of this PR, everything uses `ExternResult` so there's just one less abstraction to juggle